### PR TITLE
Add missing log to hub method failure

### LINKED ISSUE #10554: SignalR exceptions aren't logged in 3.0 preview 5
Good time of day,

I've been trying the 3.0 preview and I'm not sure where the exception logging has gone to in SignalR, does it need additional config to appear in 3.0? To make a clean test i've started 2 projects that are basically the "new project" template and used the [signalr intro tutorial](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-2.2).

One project runs on asp 2.2, click the chat button - observe the console outputs the exception
The other project runs on asp 3.0, click the chat button - you can see the exception message going across the web socket in browser dev console but nothing appears on server log

[exception-logging-issues.zip](https://github.com/aspnet/AspNetCore/files/3224146/exception-logging-issues.zip)

What configuration is needed to get server side logging for signalr action exceptions in 3.0?

### PR DESCRIPTION
**Pull request by BrennanConroy**
Fixes #10554

### CODE REVIEW

File: src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs [hunk 1]
```diff
@@ -859,10 +859,17 @@ public async Task HubMethodsAreCaseInsensitive(Type hubType)
         [InlineData(nameof(MethodHub.MethodThatYieldsFailedTask), false)]
         public async Task HubMethodCanThrowOrYieldFailedTask(string methodName, bool detailedErrors)
         {
+            var hasErrorLog = false;
             bool ExpectedErrors(WriteContext writeContext)
             {
-                return writeContext.LoggerName == "Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher" &&
+                var expected = writeContext.LoggerName == "Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher" &&
```
Comment (mikaelm12) [inline src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs]: `String.Equals()` ?

**Review by mikaelm12 — APPROVED** [review]
So that's where is was lol

**Review by analogrelay — APPROVED** [review]

**Comment by BrennanConroy** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=213200&view=logs

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2633

File: src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs (hunk 1 above)
Comment (BrennanConroy) [reply src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs]: Yeah, maybe I'll do a pass of all our tests later and do that

**Pull request merged**